crates/solver/src/settlement_submission/submitter/eden_api.rs

```diff
@@ -5,17 +5,18 @@ use crate::settlement::{Revertable, Settlement};
 use super::{
     super::submitter::{TransactionHandle, TransactionSubmitting},
     common::PrivateNetwork,
-    AdditionalTip, CancelHandle, Strategy, SubmissionLoopStatus,
+    AdditionalTip, Strategy, SubmissionLoopStatus,
 };
 use anyhow::{bail, Context, Result};
 use ethcontract::{
+    dyns::DynTransport,
     transaction::{Transaction, TransactionBuilder},
     H256,
 };
 use futures::{FutureExt, TryFutureExt};
 use reqwest::{Client, IntoUrl, Url};
 use serde::Deserialize;
-use shared::{transport::http::HttpTransport, Web3, Web3Transport};
```

Comment:
Is this really needed? I thought `Web3Transport` was an alias to `DynTransport`.

Reply:
Sure. Substituted.